arelle/plugin/validate/ESEF/__init__.py

```diff
@@ -906,10 +906,11 @@ def checkFootnote(elt, text):
             if reportedEltsNotInLb and lbType == "presentation":
                 # reported pri items excluded from having to be in pre LB
                 nsExcl = val.authParam.get("lineItemsMustBeInPreLbExclusionNsPattern")
+                nsExclPat = None
                 if nsExcl:
                     nsExclPat = re.compile(nsExcl)
                 reportedEltsNotInLb -= set(c for c in reportedEltsNotInLb 
-                                           if nsExclPat.match(c.qname.namespaceURI))
+                                           if nsExclPat and nsExclPat.match(c.qname.namespaceURI))
```

Comment:
Thank you for the contribution!

Instead of checking `nsExclPat` can you nest this under the `nsExcl` conditional?

Reply:
@ChristopheDeBernardi can you look at the above question from @austinmatherne-wk, please?

Reply:
Oops, didn't see that one.

Yes it would make more sense than what I did if the original problem was just the line not being correctly indented;
Done.